news/models.py

```diff
@@ -5,5 +5,9 @@
 class NewsData(models.Model):
     name = models.CharField(max_length=30,blank=False)
     title = models.TextField()
-    image = models.ImageField(max_length=150)
-    content = models.TextField()
\ No newline at end of file
+    image = models.CharField(max_length=200)
+    content = models.TextField()
+    time = models.CharField(max_length=80, null=False, blank=False)
+
```

Comment:
In time field don't use charField cheange to date or datetime field . check whether it works or not